### PR TITLE
LRAC-14977

### PR DESCRIPTION
**Pull request by riccardo-ferrari**

### CODE REVIEW

**Comment by riccardo-ferrari** [comment]
ci:test:sf

**Comment by riccardo-ferrari** [comment]
ci:test:relevant

**Comment by marcellustavares** [comment]
LGTM

File: modules/apps/journal/journal-taglib/src/main/resources/META-INF/resources/journal_article/page.jsp [hunk 1]
```diff
@@ -30,7 +30,7 @@ String wrapperCssClass = (String)request.getAttribute("liferay-journal:journal-a
 		</div>
 	</c:when>
 	<c:otherwise>
-		<div class="journal-content-article <%= Validator.isNotNull(wrapperCssClass) ? wrapperCssClass : StringPool.BLANK %>" <%= dataAnalyticsTrackingEnabled ? String.format("data-analytics-asset-id=\"%s\" data-analytics-asset-title=\"%s\" data-analytics-asset-type=\"web-content\"", articleDisplay.getArticleId(), HtmlUtil.escapeAttribute(articleDisplay.getTitle())) : "" %>>
+		<div class="journal-content-article <%= Validator.isNotNull(wrapperCssClass) ? wrapperCssClass : StringPool.BLANK %>" <%= dataAnalyticsTrackingEnabled ? String.format("data-analytics-asset-id=\"%s\" data-analytics-asset-title=\"%s\" data-analytics-asset-type=\"web-content\" data-analytics-asset-resource-prim-key=\"%s\"", articleDisplay.getArticleId(), HtmlUtil.escapeAttribute(articleDisplay.getTitle()), articleDisplay.getResourcePrimKey()) : "" %>>
```
Comment (marcellustavares) [inline modules/apps/journal/journal-taglib/src/main/resources/META-INF/resources/journal_article/page.jsp:33]: can we rename  to  `web-content-resource-pk`

**Review by marcellustavares — CHANGES_REQUESTED** [review]
@riccardo-ferrari, please see my comments above.

**Comment by riccardo-ferrari** [comment]
ci:test:sf

**Comment by riccardo-ferrari** [comment]
ci:test:relevant

**Review by interaminense — APPROVED** [review]
LGTM! Thanks @riccardo-ferrari

**Comment by marcellustavares** [comment]
ci:forward:force

**Pull request closed without merging**